app/views/partials/a-propos.html

```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <h1>A propos</h1>
+  <h1 tabindex="-1">A propos</h1>
```

Comment:
Can't these be added dynamically through JS to avoid repeating (and forgetting to add) the attribute on each page?